portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -80,8 +88,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentEntity);
 
-    // Redirect to home after comment is submitted
-    response.sendRedirect("/");
+    // Refresh after comment is submitted
+    response.sendRedirect("/comments.html");
+  }
+
+  /*
+   * Gets the number of comments requested based on string parameter.
+   * Defaults to MAX_VALUE if argument is null or empty.
+   * @return requested number of comments.
+   */
+  private int getNumberOfComments(String numberOfCommentsString) {
+    int numComments;
```

Comment:
Instead of caching this value in a variable you can simply return it at each line:

if (...) {
   return Integer.MAX_VALUE;
}
return Integer.parseInt(...)

Reply:
Good point, made the change.